### PR TITLE
Update configure-server-exclusions-microsoft-defender-antivirus.md

### PR DESCRIPTION
**Pull request by msbemba**
Added more information to the note stating that automatic exclusions apply only to Windows Server 2016 and later. These exclusions are not visible in the Windows Security app and in PowerShell.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/configure-server-exclusions-microsoft-defender-antivirus.md [hunk 1]
```diff
@@ -71,7 +71,9 @@ Because Microsoft Defender Antivirus is built in, it does not require exclusions
 Operating system exclusions and server role exclusions do not appear in the standard exclusion lists that are shown in the [Windows Security app](microsoft-defender-security-center-antivirus.md).
 
 > [!NOTE]
-> Automatic exclusions for server roles and operating system files do not apply to Windows Server 2012. Automatic exclusions can apply if your servers running Windows Server 2012 R2 are onboarded to Defender for Endpoint. (See [Onboard Windows servers to the Microsoft Defender for Endpoint service](configure-server-endpoints.md).)
+> Automatic exclusions apply only to Windows Server 2016 and later. These exclusions are not visible in the Windows Security app and in PowerShell.
+> Automatic exclusions can apply if your servers running Windows Server 2012 R2 are onboarded to Defender for Endpoint. (See [Onboard Windows servers to the Microsoft Defender for Endpoint service](configure-server-endpoints.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-server-exclusions-microsoft-defender-antivirus.md]:
```suggestion
> Automatic exclusions can apply if your servers running Windows Server 2012 R2 are onboarded to Defender for Endpoint. For more information, see [Onboard Windows servers to the Microsoft Defender for Endpoint service](configure-server-endpoints.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/security/defender-endpoint/configure-server-exclusions-microsoft-defender-antivirus.md (hunk 1 above)
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-server-exclusions-microsoft-defender-antivirus.md]:
```suggestion
> Automatic exclusions apply only to Windows Server 2016 and later. These exclusions are not visible in the Windows Security app and in PowerShell.
>
> Automatic exclusions can apply if your servers running Windows Server 2012 R2 are onboarded to Defender for Endpoint. (See [Onboard Windows servers to the Microsoft Defender for Endpoint service](configure-server-endpoints.md).
>
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@denisebmsft Please merge this PR. Thanks

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba @JohanFreelancer9

**Pull request merged**